docs/_pages/0005-blog.md

```diff
@@ -24,4 +24,4 @@ next_url: /docs/changelog/
 
 In the ever-evolving landscape of cloud-native applications, secrets management
 is critical to ensuring sensitive information’s security and integrity. While
-several solutions are... [Read More](/blog/keep-your-secrets/)
+several solutions are… [Read More](/blog/keep-your-secrets/)
```

Comment:
Thinking…

Reply:
Yes, and no.

Two reasons:
1. Excerpt is plain-text; we can have rich formatting in this one.
2. This PR does not include a way to inject excerpt into the post; it injects it into the main listing of posts here https://vsecm.com/docs/blog/
3. If we make excerpt rich-formatted then it will complicate things because the code truncates stuff after a certain length, and it will miscalculate it becuse of the extra baggage the the tags bring.


Well that made three reasons :D — We can create an issue and explore it further though after we merge this.